packages/paste-types/package.json

```diff
@@ -23,7 +23,7 @@
     "tsc": "tsc"
   },
   "dependencies": {
-    "@types/lodash": "^4.17.19",
+    "@types/lodash": "^4.14.182",
```

Comment:
Question: the I noticed the rest of the `package.json`s use `^4.7.21`. Is there a reason why this one is different?